### ISSUE TITLE
Installation disabling SELinux

### ISSUE THREAD

**Issue by satkunas** [inverse-inc/packetfence#3703]
The installation guide states:

> Regarding SELinux or AppArmor, even if they may be wanted by some organizations, PacketFence will not work properly if SELinux or AppArmor are enabled. You will need to explicitly disable SELinux from the /etc/selinux/config file and reboot the machine. 

But it doesn't actually state (in the guide) what the line needs to be changed to:
`SELINUX=disabled`

Although the options are listed in /etc/selinux/config, it would be beneficial to repeat the required line in the installation doc.

**Comment by nqb** [comment]
Agree.

**Comment by nqb** [comment]
Maybe we can cut/paste section from: https://github.com/inverse-inc/packetfence/blob/b2a85bad7150335ae3c98841f688d0ba700232e0/docs/pki/packetfence.asciidoc#preparing-to-install-in-standalone-mode and do a reference to this section in pki/packetfence.asciidoc.

**Comment by satkunas** [comment]
@nqb This works for me....

Furthermore, in reference to your link above, there's an indentation formatting error in this section that should also be adjusted. "Change the line:" and "to" (and their related commands) should have the same indentation. They currently differ...

![screenshot from 2018-10-24 15-06-39](https://user-images.githubusercontent.com/3904468/47454983-9d613c00-d79e-11e8-912a-dca9629c475d.png)

**Comment by julsemaan** [comment]
@satkunas

Want to take care of the fix for that issue ?

**Comment by Emil-G** [comment]
Why would you disable selinux? 

> On Red Hat-based systems 
    - Disable firewall
    - Disable SELinux

.. Same for AppArmor
> 

Is this a joke in the installation guide? Why can't a software claiming to be a security solution not support fundamental os security?

**Comment by fdurand** [comment]
@Emil-G , really cool that you want to take care of that, we waiting the PR.

**Comment by julsemaan** [comment]
Yes, considering the very small amount of services PacketFence has and inter-connectivity between them, that's definitely an easy job to add SELinux on PacketFence.

I'm pretty sure all the big names in NAC have SELinux and AppArmor enabled in their proprietary Linux distro on which they built their NAC.

Come on PF team, get it together...

**Comment by extrafu** [comment]
I have an other suggestion:  let's port PacketFence to OpenBSD because it's the only secure OS out there!
